assets/xml/objects/object_and.xml

```diff
@@ -1,79 +1,80 @@
+        <DList Name="gAndTrainDL" Offset="0x4670" />
+        <DList Name="gAndCalvesDL" Offset="0x4708" />
+        <DList Name="gAndShinsDL" Offset="0x47F0" />
+        <DList Name="gAndThighsDL" Offset="0x4998" />
+        <DList Name="gAndPelvisDL" Offset="0x4BE0" />
+        <DList Name="gAndLowerVeilDL" Offset="0x4D90" />
+        <DList Name="gAndUpperVeilDL" Offset="0x4E50" />
+        <DList Name="gAndVeilBaseDL" Offset="0x4F20" />
+        <DList Name="gAndHeadDL" Offset="0x5038" />
+        <DList Name="gAndBouquetDL" Offset="0x56C0" />
+        <DList Name="gAndCenterLeavesTipDL" Offset="0x5888" />
+        <DList Name="gAndCenterLeavesDL" Offset="0x5938" />
+        <DList Name="gAndRightLeavesTipDL" Offset="0x59C8" />
+        <DList Name="gAndRightLeavesDL" Offset="0x5A80" />
+        <DList Name="gAndLeftLeavesTipDL" Offset="0x5BA8" />
+        <DList Name="gAndLeftLeavesDL" Offset="0x5C60" />
+        <DList Name="gAndRightHandDL" Offset="0x5D80" />
+        <DList Name="gAndRightForearmDL" Offset="0x5F10" />
+        <DList Name="gAndRightUpperArmDL" Offset="0x5FF8" />
+        <DList Name="gAndLeftHandDL" Offset="0x6240" />
+        <DList Name="gAndLeftForearmDL" Offset="0x63D8" />
+        <DList Name="gAndLeftUpperArmDL" Offset="0x64C0" />
+        <DList Name="gAndSymbolDL" Offset="0x6708" />
+        <DList Name="gAndTorsoDL" Offset="0x67E0" />
+        <Texture Name="gAndSkinShadowTex" OutName="and_skin_shadow" Format="rgba16" Width="8" Height="8" Offset="0x6C30" />
+        <Texture Name="gAndHairTex" OutName="and_hair" Format="rgba16" Width="8" Height="16" Offset="0x6CB0" />
+        <Texture Name="gAndEarTex" OutName="and_ear" Format="rgba16" Width="16" Height="16" Offset="0x6DB0" />
+        <Texture Name="gAndFingersTex" OutName="and_fingers" Format="rgba16" Width="16" Height="16" Offset="0x6FB0" />
+        <Texture Name="gAndDressGradientTex" OutName="and_dress_gradient" Format="i8" Width="8" Height="8" Offset="0x71B0" />
+        <Texture Name="gAndDressSymbolTex" OutName="and_dress_symbol" Format="i8" Width="32" Height="32" Offset="0x71F0" />
+        <Texture Name="gAndPearlTex" OutName="and_pearl" Format="rgba16" Width="16" Height="16" Offset="0x75F0" />
+        <Texture Name="gAndLeavesTex" OutName="and_leaves" Format="rgba16" Width="32" Height="16" Offset="0x77F0" />
+        <Texture Name="gAndFlowerTex" OutName="and_flower" Format="rgba16" Width="16" Height="16" Offset="0x7BF0" />
+        <Texture Name="gAndEyeOpenTex" OutName="and_eye_open" Format="rgba16" Width="32" Height="32" Offset="0x7DF0" />
+        <Texture Name="gAndEyeClosingTex" OutName="and_eye_closing" Format="rgba16" Width="32" Height="32" Offset="0x85F0" />
+        <Texture Name="gAndEyeClosedTex" OutName="and_eye_closed" Format="rgba16" Width="32" Height="32" Offset="0x8DF0" />
+        <Texture Name="gAndEyeOpeningTex" OutName="and_eye_opening" Format="rgba16" Width="32" Height="32" Offset="0x95F0" />
```

Comment:
Normally we use `EyeHalfTex` instead of words like `opening` and `closing` for eyes that are half-way open, but because there's 2 "half", maybe opening/closing is fine. Anyone else have thoughts?

Reply:
EyeHalf is only appropriate for 3 state eye animations. From what I've seen of it, 4 state eye animations clearly have very different artwork between closing and opening textures, necessitating the distinctive names.